specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/ManagedEnvironments.json

```diff
@@ -859,6 +859,10 @@
           "$ref": "#/definitions/EnvironmentSkuProperties",
           "description": "SKU properties of the Environment."
         },
+        "kind": {
+          "type": "string",
+          "description": "Kind of the Environment."
+        },
```

Comment:
Should we update the sample files to include this property?

Reply:
Yes, we can. But as it is optional, it is not required to add in examples.